packages/api/tests/models/test-executions-model.js

```diff
@@ -47,6 +48,7 @@ test('generateRecord() returns the correct record in the basic case', (t) => {
   const expectedRecord = {
     name: executionName,
     arn: executionArn,
+    cumulusVersion: '1.2.3',
```

Comment:
non-pr-blocking nit: term 'basic-case' assumes knowledge in the test description